### PR TITLE
Unit tests for metasync's transport

### PR DESCRIPTION
**Pull request by usmanong**
@alex-aizman @VladimirMarkelov @liangdrew @sasanap

### CODE REVIEW

File: dfc/metasync_test.go [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+
+	if m[i].id != m[j].id {
+		return m[i].id < m[j].id
+	}
+
+	return m[i].cnt < m[j].cnt
+}
+
+// newPrimay returns a proxy runner after initializing the fields that are needed by this test
+func newPrimay() *proxyrunner {
```
Comment (VladimirMarkelov) [inline dfc/metasync_test.go]: Typo in 'newPrimary'?

Comment (usmanong) [reply dfc/metasync_test.go]: yup, thanks

**Comment by usmanong** [comment]
diff updated

**Comment by usmanong** [comment]
thank you

**Pull request closed without merging**